jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/TextField.java

```diff
@@ -22,96 +19,157 @@
-    private String prefix = ".v-text-field__prefix";
-    private String suffix = ".v-text-field__suffix";
-    private String icon = ".v-icon";
-
+    private static final String INPUT = ".//input|.//textarea";
+    private static final String SLOT = ".v-input__slot";
+    private static final String MESSAGE = ".v-messages__message";
+    private static final String COUNTER = ".v-counter";
+    private static final String PREFIX = ".v-text-field__prefix";
+    private static final String SUFFIX = ".v-text-field__suffix";
+    private static final String ICON = ".v-icon";
+
+    /** Get the text input field element inside the TextField.
+     *
+     * @return text input field as {@link UIElement}
+     */
     @JDIAction("Get '{name}' text input field")
     public UIElement textInputField() {
-        return find(input);
+        return find(INPUT);
     }
 
+    /** Get the icon element inside the TextField.
+     *
+     * @return icon as {@link UIElement}
+     */
     @JDIAction("Get '{name}' icon")
     public UIElement icon() {
-        return find(icon);
+        return find(ICON);
     }
 
+    /** Get the slot of the TextField.
+     *
+     * @return message as {@link UIElement}
+     */
     @JDIAction("Get '{name}' slot")
     public UIElement slot() {
-        return find(slot);
+        return find(SLOT);
     }
 
+    /** Get the message element of the TextField.
+     *
+     * @return message as {@link UIElement}
+     */
     @JDIAction("Get '{name}' message")
     public UIElement message() {
-        return find(message);
+        return find(MESSAGE);
     }
 
+    /** Get the counter of the TextField.
+     *
+     * @return message as {@link UIElement}
+     */
     @JDIAction("Get '{name}' counter")
     public UIElement counter() {
-        return find(counter);
+        return find(COUNTER);
     }
 
+    /** Get the prefix of the TextField.
+     *
+     * @return message as {@link UIElement}
+     */
     @JDIAction("Get '{name}' prefix")
     public UIElement prefix() {
-        return find(prefix);
+        return find(PREFIX);
     }
 
+    /** Get the suffix of the TextField.
+     *
+     * @return message as {@link UIElement}
+     */
     @JDIAction("Get '{name}' suffix")
     public UIElement suffix() {
-        return find(suffix);
+        return find(SUFFIX);
     }
 
+
+    /** Get the text of TextField.
+     *
+     * @return message as {@link String}
+     */
     @Override
     @JDIAction("Get '{name}' text")
     public String getText() {
         return textInputField().getText();
     }
 
+    /** Get the text type of TextField. The type is the attribute 'type' of the input field inside the TextField.
+     *
+     * @return message as {@link String}
+     */
     @JDIAction("Get '{name}' text type")
     public String getTextType() {
         return textInputField().getAttribute("type");
     }
 
+    /** Get the label of the TextField.
+     *
+     * @return label as {@link Label}
+     */
```

Comment:
If we override the method, then there is no need to write a JavaDoc unless there are some major differences from the parent version.